american-medical-association-alphabetical.csl

```diff
@@ -211,9 +213,19 @@
           <group prefix=" " suffix=".">
             <text variable="container-title" font-style="italic" form="short"/>
             <group delimiter=";" prefix=" ">
-              <date variable="issued">
-                <date-part name="year"/>
-              </date>
+             <choose>
```

Comment:
I don't agree with your reading of the manual here. I think, if you look at the example, they're referring to journal articles with neither volume nor issue. So in line 217 you should have ```variable="issue volume"``` instead.